server/core/peer.go

```diff
@@ -80,16 +80,23 @@ type PeerInfo struct {
 	writtenKeys  uint64
 	readBytes    uint64
 	readKeys     uint64
+	intervals    uint64
+	regionInfo   *RegionInfo
```

Comment:
It is wired to contain RegionInfo in PeerInfo.

Reply:
removed